addons/l10n_ar_withholding/wizards/l10n_ar_payment_register_withholding.py

```diff
@@ -0,0 +1,69 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import logging
+
+from odoo import models, fields, api
+
+_logger = logging.getLogger(__name__)
+
+
+class l10nArPaymentRegisterWithholding(models.TransientModel):
+    _name = 'l10n_ar.payment.register.withholding'
+    _description = 'Payment register withholding lines'
+
+    payment_register_id = fields.Many2one('account.payment.register', required=True, ondelete='cascade')
+    company_id = fields.Many2one(related='payment_register_id.company_id')
+    currency_id = fields.Many2one(related='payment_register_id.currency_id')
+    name = fields.Char(string='Number')
+    tax_id = fields.Many2one('account.tax', check_company=True,  required=True)
```

Comment:
```suggestion
    tax_id = fields.Many2one('account.tax', check_company=True, required=True)
```

Reply:
could use a label...  And does not the class require check_company_auto if we use check_company here?